packages/coins/src/tron/index.ts

```diff
@@ -0,0 +1,38 @@
+import coinList from './coins';
+
+import { ICoinInfo, coinFamiliesMap } from '../types';
+
+type TronFamily = typeof coinFamiliesMap.tron;
+
+export interface ITronCoinInfo extends ICoinInfo {
+  family: TronFamily;
+  apiCoinType: string;
```

Comment:
I think `apiCoinType` won't be needed in case of tron

Reply:
done